controller/src/beerocks/cli/beerocks_cli.cpp

```diff
@@ -91,9 +91,18 @@ bool cli::parsedCommandArgumentsLinux(std::string line)
         resetArguments();
         return false;
     }
+
     const commandInfo &info = commandIt->second;
     commandNumOfArgs        = commandTokens.size() - 1;
 
+    int i                = 0;
+    bool late_processing = false;
+    auto first_arg       = std::next(commandTokens.begin());
+    if (first_arg != commandTokens.end() && (*first_arg)[0] == '"') {
+        late_processing  = true;
```

Comment:
This is a pretty stupid way of handling quotes - but OK, it was already like that, and this simple change indeed fixes it. Simple is better!

Reply:
Why do you think it is stupid? how would you do it instead?

Reply:
Sorry, stupid was the wrong word, I meant "naive", because it only supports putting all arguments together between quotes, i.e.
```
command "foo bar baz"
```
and not something like
```
command "foo bar" baz
comman2 foo "bar baz"
```

Reply:
Yes, there is no other command the has spaces in a single argument. The WFA-CA command is the only command that has it and also, all the text it the command should be read as a single argument because it is not processed in the cli.
So yeah it was the simple naive solution because we didn't want anything more back then.